package.json

```diff
@@ -5,7 +5,7 @@
   "description": "A to do list app",
   "main": "index.js",
   "scripts": {
-    "build": "webpack",
+    "build": "webpack --mode production",
     "dev": "webpack serve"
   },
```

Comment:
- [ ] [REQUIRED] Good job so far! Your project is working fine but there is one script left to run it. Please correct it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/222496840-38958044-3658-417f-9ee2-976ca222d4a0.png)

- `[HINT]` Please paste this `script` after `line 9` 👍🏼 
```json
"start": "webpack serve --open"
```